examples/Receiver_Response.py

```diff
@@ -0,0 +1,247 @@
+""" Calculate and Fit Receiver Response to Sinc
+===============================================
+
+Two files are required for the following example:
+
+    File1 contains the analytic signal acquired on the GDS oscilloscope
+    directly output from the AFG output. Each node pertains to signal with a
+    different frequency (in kHz) which are fit to a complex function to extract
+    the :math:`V_{pp}` (in μV).
+    File2 contains the quadrature signal acquired on the receiver when the same
+    signal of File1 is injected into it. Each node pertains to signal with a
+    different frequency (in Hz) which are converted to a PSD. The amplitude
+    (in dg) is calculated from the peak of the convolved PSD.
+
+    The receiver response is then the ratio of :math:`dg(\\nu)` to
+    :math:`μV(\\nu)` which is fit to an absolute sinc function.
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+from numpy import r_, pi
+import pyspecdata as psd
+from sympy import symbols
+import sympy as sp
+import re
+
+Dnu_name = r"$\Delta\nu$"
+nu_direct = r"$\nu_{direct}$"
+lambda_G = 0.4e3  # Width for Gaussian convolution
+data_dir = "ODNP_NMR_comp/noise_tests"
+file1 = "240123_10mV_AFG_GDS_5mV_100MSPS_analytic.h5"
+file2 = "240117_afg_sc_10mV_3p9kHz_zoom.h5"
+
+
+# {{{ Function to return the nodenames, test signal frequencies, and an
+#     empty nddata to drop calculated dg and V into
+def make_ndData(thisfile, nodes_in_kHz=True):
+    """Determine the nodenames of the file and sort them according to the
+    frequency of the test signal. Based on these nodenames a list of the
+    frequencies as integers are made. Anempty NDData in the shape of the
+    acquired data is created with an extra dimension 'nu_test' that
+    the sorted frequencies are placed inside that dimension.
```

Comment:
this is just too wordy.  Don't overdramatize.  You look at the nodes in the hdf5 file, and you put them in an ordered list and use it to label the axis coordinates of an empty nddata that you also return.